crates/sncast/src/helpers/constants.rs

```diff
@@ -13,8 +13,7 @@ inputs = []
 "#;
 
 pub const UDC_ADDRESS: &str = "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf";
-pub const OZ_CLASS_HASH: &str =
-    "0x058d97f7d76e78f44905cc30cb65b91ea49a4b908a76703c54197bca90f81773";
+pub const OZ_CLASS_HASH: &str = "0x4c6d6cf894f8bc96bb9c525e6853e5483177841f7388f74a46cfda6f028c755";
```

Comment:
Note: This class hash has been declared on the mainnet, sepolia and goerli. It was taken from the Slack conversation with SW and is also used in [starkli](https://github.com/xJonathanLEI/starkli/blob/master/src/account.rs#L65).